test/unit/testingNodeIpPort.ts

```diff
@@ -0,0 +1,55 @@
+import net from 'net'
```

Comment:
This is not the correct way to write unit tests. Please take shardeum, shardus-core as example and write tests accordingly.

Reply:
+ I think the filename name should be like this: {FILENAME}.test.ts